docs/install/install-aur.md

```diff
@@ -6,6 +6,37 @@ sidebar_position: 4
 
 # Installing Vignette using AUR
 
-something
+If you wish to install Vignette on your Arch Linux computer, you may use any AUR helper of you choice. The package is [`vignette-git`](https://aur.archlinux.org/packages/vignette-git/). In the example below, I will be using yay.
+
+
+
+## Installation process
+
+In order to install Vignette using yay, please run the following command: 
+
+`yay vignette-git`
+
+
+
+If more than one package appears you should find the right package, which should be listed as `aur/vignette-git`. In this example, that package is number `1`.
+
+![](/img/install/yay-terminal-window.webp)
+
+
+
+:::note 
+
+You may be required to type your password into the terminal when prompted.
+
+:::
+
+
+
+
+
+## Welcome to Vignette!
+
+Please visit the UI Overview page [here](/client/overview.md) or from the sidebar in order to make yourself more familiar with the interface and features of the app. Thank you for chosing Vignette!
```

Comment:
they aren't required to read it if they wanna go on their own.

```suggestion
You can now visit the UI Overview page [here](/client/overview.md) in order to make yourself more familiar with the interface and features of the app.

Thank you for installing Vignette!
```

Reply:
please sounds more friendly, and the sidebar thing is a good backup

Reply:
"you can now do this" sounds way better than "please do this". You're not begging them

Reply:
if you cant install an AUR package you most likely cant figure the UI out yourself, so being more pushy seems right

Reply:
i mean... people don't read this because they don't know how to do it. Maybe they don't know what the package name is, or if there even is an AUR package.

Reply:
yay --query is a thing people use

Reply:
i didn't even know it existed. Well... this is getting off-topic. Since we can't decide, let's get a third opinion

Reply:
@sr229 could you please weigh in?